api/pin.js

```diff
@@ -14,6 +14,8 @@ module.exports = async (req, res) => {
   } = req.query;
 
   let repoData;
+  
+  res.setHeader("Cache-Control", "public, max-age=300");
```

Comment:
I would suggest to set the cache to 30 mins, five minutes is nothing
```suggestion
  res.setHeader("Cache-Control", "public, max-age=1800");
```

Reply:
How about 15? still people would have frequent updates :thinking:

Reply:
I would also set it a bit higher, there is no real need to refresh each time. I would even consider 1h.

Reply:
Also, I would like to change the magic number to a constant, to make it clearer.

Reply:
I would probably add that number to env variable so that if i want to change it i dont have to change the code

Reply:
Nice, that would solve it, and make it easier to change in the future 😃